TrafficCapture/trafficReplayer/README.md

```diff
@@ -76,12 +76,57 @@ target URI.
 
 ## Transformations
 
-Transformations are performed via a simple class defined by
-[IJsonTransformer](../trafficReplayer/src/main/java/org/opensearch/migrations/transform/IJsonTransformer.java). Currently,
-this class uses [JOLT](https://github.com/bazaarvoice/jolt) to perform transforms that are composed of modular
-operations that are defined in the [resources](../trafficReplayer/src/main/resources/jolt/operations) associated with
-the package. Future work will include adding more JSON transformations and other potential JSON transformation tools
-(like [JMESPath](https://jmespath.org/)).
+Transformations are performed via a simple interface defined by
+[IJsonTransformer](../replayerPlugins/jsonMessageTransformers/jsonMessageTransformerInterface/src/main/java/org/opensearch/migrations/transform/IJsonTransformer.java) ('transformer').  They are loaded dynamically and are designed to allow for easy extension
+of the TrafficReplayer to support a diverse set of needs.
```

Comment:
I would indicate that this is key to supporting migration of older workloads or workloads when source and target differ with breaking API changes.

Reply:
With the offloading of Transformers, the TrafficReplayer is now pretty much a completely workload agnostic tool.  I'd rather keep its own docs agnostic as well.